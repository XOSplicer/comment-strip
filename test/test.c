#include <stdio.h>
/**
* multi line comment
*/
int main() {
    //single line comment
    printf("//no comment\"");
}
