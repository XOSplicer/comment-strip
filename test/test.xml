<!-- some comment -->
<tag>
<!-- multi
line
comment -->
</tag><!----><b />
<t value="string<!-- no comment -->" />
<!--/t-->

