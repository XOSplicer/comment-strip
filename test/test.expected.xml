<tag>
</tag><b />
<t value="string<!-- no comment -->" />
